recipes/odm2rest/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.0a" %}
+
+package:
+  name: odm2rest
+  version: {{ version }}
+
+source:
+  # not useing the jinja variable for the version b/c the release is not pep440
+  # see https://www.python.org/dev/peps/pep-0440/#id25
+  url: https://github.com/ODM2/ODM2RESTfulWebServices/archive/v0.1.0-alpha.tar.gz
+  sha256: f79d94dd58e7ed0e223049067bfc9e1d98234bca9ddf65bfd9eb62ff14d2a8bf
+
+build:
+  number: 0
+  skip: True  # [py3k]
```

Comment:
Based on the qualifiers this is not Python 3 ready yet, right?

Reply:
Correct